packages/docs/static/themes/cmsgov-theme.css

```diff
@@ -493,7 +493,6 @@
 --typography-heading-lg__font-weight: 700;
 --typography-heading-md__font-weight: 700;
 --typography-heading-sm__font-weight: 400;
---typography-heading-base__font-size: 1rem;
```

Comment:
Huh, is `main` not up-to-date?

Reply:
I just [put  up a PR to fix it](https://github.com/CMSgov/design-system/pull/2812)